draft-ietf-dnsop-svcb-https.md

```diff
@@ -1409,6 +1387,55 @@ and others for their feedback and suggestions on this draft.
 
 --- back
 
+# Decoding text in zone files {#decoding}
+
+DNS zone files are capable of representing arbitrary octet sequences in
+basic ASCII text, using various delimiters and encodings.  The algorithm
+for decoding these character-strings is defined in Section 5.1 of {{RFC1035}}.
+Here we summarize the allowed input to that algorithm, using ABNF:
+
+    ; non-special is VCHAR minus DQUOTE, ";", "(", ")", and "\".
+    non-special = %x21 / %x23-27 / %x2A-3A / %x3C-5B / %x5D-7E
+    ; non-digit is VCHAR minus DIGIT
+    non-digit   = %x21-2F / %x3A-7E
+    ; dec-octet is a number 0-255 as a three-digit decimal number.
+    dec-octet   = ( "0" / "1" ) 2DIGIT /
+                  "2" ( ( %x30-34 DIGIT ) / ( "5" %x30-35 ) )
+    escaped     = "\" ( non-digit / dec-octet )
+    contiguous  = 1*( non-special / escaped )
+    quoted      = DQUOTE *( contiguous / ( ["\"] WSP ) ) DQUOTE
+    char-string = contiguous / quoted
+
+The decoding algorithm allows `char-string` to represent any `*OCTET`.
+In this document, this algorithm is referred to as "character-string decoding".
+The algorithm is the same as used by `<character-string>` in RFC 1035,
+although the output length in this document is not limited to 255 octets.
+
+## Decoding a value list {#value-list}
+
```

Comment:
It took me a few times reading this to really understand it.  Not sure ways to clarify?
Corner cases:
* What about \\44 ?  (encoded comma)
* DQUOTE
Perhaps have an example which shows how these interact?
For example, that:
"part1,part2","part3" 
still gets decoded as all three values?
And that "part1\\44part2,part3" gets decoded as two values?

Reply:
OK, clarified relationship with \044 and quoting.

> "part1,part2","part3"

This is not a valid `char-string` so it's excluded.

I admit that this description isn't perfect, but there is no clear syntax definition for zone files so I don't think we should aim for extreme precision.